examples/datasets/plot_limo_data.py

```diff
@@ -49,7 +49,7 @@
 print(__doc__)
 
 # fetch data from subject 2 and interpolate missing channels
-limo_epochs = limo.load_data(subject=2)
+limo_epochs = limo.load_data(subject=2, update_path=True)
```

Comment:
update_path is not set explicitly in any example AFAIK

is this necessary here?

Reply:
no not really. The update is done automatically. I'll delete it from the files calling ``limo.load_data``